tools/boost-builder/upload_boosts.zsh

```diff
@@ -0,0 +1,10 @@
+#!/bin/zsh
```

Comment:
No - upload_boosts was simple enough that I never had a problem that prompted me to add it. Added.